x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2707,6 +2709,13 @@ export class RulesClient {
       actions: actions ? this.injectReferencesIntoActions(id, actions, references || []) : [],
       params: this.injectReferencesIntoParams(id, ruleType, params, references || []) as Params,
       ...(includeSnoozeSchedule ? { snoozeSchedule: snoozeScheduleDates } : {}),
+      ...(includeSnoozeSchedule
+        ? {
+            activeSnoozes: getActiveSnoozes({ snoozeSchedule })
+              ?.filter((s) => Boolean(s.id))
```

Comment:
nitpick: do we need to filter again? I saw that we're already filtering in getActionSnoozes

Reply:
Yeah we need to filter out any snooze with an `undefined` id (which would be a relative snooze). `activeSnoozes` is supposed to only be for displaying active scheduled snoozes.

Now that I type that out though, maybe that filtering should actually be happening at the UI level.

Reply:
Turns out other uses of `getActiveSnoozes` only need the recurring snoozes as well so I just altered the function. ID filtering will now happen within the renamed `getActiveScheduledSnoozes`